modules/appeals_api/spec/docs/supplemental_claims/v0_spec.rb

```diff
@@ -243,26 +243,22 @@ def swagger_doc
         Evidence may be uploaded up to 7 days from the 'created_at' date of the associated Supplemental Claim via 'supplemental_claims/evidence_submissions'.
       DESC
 
-      parameter name: :sc_uuid,
-                in: :query,
-                type: :string,
-                required: true,
-                description: 'Associated Supplemental Claim UUID',
-                example: '7efd87fc-fac1-4851-a4dd-b9aa2533f57f'
-
-      parameter AppealsApi::SwaggerSharedComponents.header_params[:veteran_ssn_header]
-      let(:'X-VA-SSN') { '123456789' }
-
       security DocHelpers.oauth_security_config(scopes)
+
+      consumes 'application/json'
       produces 'application/json'
 
-      response '202', 'Accepted. Location generated' do
-        let(:sc_uuid) { FactoryBot.create(:supplemental_claim_v0).id }
+      parameter name: :sc_es_body, in: :body, schema: { '$ref' => '#/components/schemas/scEvidenceSubmissionCreate' }
 
+      let(:ssn) { '123456789' }
+      let(:sc_uuid) { FactoryBot.create(:supplemental_claim_v0).id }
+      let(:sc_es_body) { { ssn:, sc_uuid: } }
+
+      response '202', 'Accepted. Location generated' do
         schema '$ref' => '#/components/schemas/scEvidenceSubmissionResponse'
 
         before do
-          allow_any_instance_of(VBADocuments::UploadSubmission).to receive(:get_location).and_return(+'http://some.fakesite.com/path/uuid')
+          allow_any_instance_of(VBADocuments::UploadSubmission).to receive(:get_location).and_return(+'http://path.to.upload/location/uuid')
```

Comment:
I changed this example path because although "fakesite.com" is appropriate for tests, it seemed odd to me for use in the docs - happy to adjust this if anyone has other ideas, though

Reply:
I like this a lot better, too!